integration_tests/e2e/arrivals.cy.ts

```diff
@@ -0,0 +1,53 @@
+import premisesFactory from '../../server/testutils/factories/premises'
+import arrivalFactory from '../../server/testutils/factories/arrival'
+
+import ArrivalCreatePage from '../pages/arrivalCreate'
+import PremisesShowPage from '../pages/premisesShow'
+
+context('Arrivals', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+  })
+
+  it('creates an arrival', () => {
+    // Given I am logged in
+    cy.signIn()
+
+    // And I have a booking for a premises
+    const premises = premisesFactory.build()
+    const bookingId = 'some-uuid'
+    const arrival = arrivalFactory.build()
+
+    cy.task('subSinglePremises', premises)
```

Comment:
haven't had chance to look through this properly yet but this should be `stubSinglePremises`

Reply:
I also can't find where the task is defined

Reply:
Was a typo here https://github.com/ministryofjustice/approved-premises-ui/blob/02b0f0d31d754f78b58f81cd7d903f4284f3d25e/integration_tests/mockApis/premises.ts#L21 All fixed now!